en/services/opendroneid.md

```diff
@@ -79,80 +107,74 @@ There is no need for this component to listen to drone ID MAVLink messages.
+For UASs that desire to listen to other UA's information, it would be desirable to include receivers for all three methods, in order to maximize the possibility of detecting all other surrounding UAs.
 
 
-## Combining data from other UAs when receiving drone ID data
+## Combining data from other UAs when receiving drone ID data {#combining}
 
-For Drone ID data that is received from other UAs, the data message itself does not always identify exactly which UA the data originated from.
+For Drone ID data that is received from other UAs, the data of the message itself does not always identify exactly which UA the data originated from.
 E.g. for data received via Bluetooth Legacy Advertising (Bluetooth 4.x), many of the received messages will not contain the unique serial number/ID of the UA that transmitted the data, due to the severe size limitation imposed by Legacy Advertising where only one 25 byte message can be transmitted in one radio burst.
 The MAC address of the Bluetooth transmitter is the only way to associate these messages to the same UA.
-For Bluetooth 5.x and WiFi, it is possible that the same can happen, although this is less likely since message packs are supposed to be used.
+For Bluetooth 5.x and Wi-Fi, it is possible that the same can happen in certain specific situations (e.g. sending large amount of authentication data), although for the majority of normal usage this is unlikely since the use of message packs is mandated.
 For data received via internet, the data packet will always contain the unique serial number/ID of the originating UA but no associated MAC address.
 
-In order to allow e.g. a DAA component to sort and identify which UA each data message has originated from, the receiver components must add either the MAC address or the ID number associated with the UA that originated the data message to the MAVLink message, before sending it on the internal UAS MAVLink network.
+In order to allow e.g. a DAA component to sort and identify which UA each data message has originated from, the receiver components must add, to the MAVLink message, either the MAC address or the ID number associated with the UA that originated the data message, before sending it on the internal UAS MAVLink network.
 This information must be added in the `id_or_mac` field of each MAVLink message.
 
 The serial/ID is copied directly from the `uas_id` field with NULLs in the unused portion.
 The MAC address must be entered in ASCII format with NULLs in the unused portion.
 Any separation characters must be removed.
 E.g. "30-65-EC-6F-C4-58" or "30:65:EC:6F:C4:58" must be represented as the ASCII string "3065EC6FC458".
-When not used for the above purpose, the `id_or_mac` field should be filled with zeroes.
+When not used for the above purpose, the `id_or_mac` field must be filled with NULLs.
```

Comment:
Probably a stupid question, but what is a `NULL`? I.e. what would I put in the field?

Reply:
The value 0x00. Since line 173 says that this field must have ASCII characters, I thought specifying zeroes was ambiguous, since it was not clear whether you should put 0x00 or 0x30. 
It makes more sense to treat this as a C-string full of NULL termination characters (and this makes it sufficient to do a memset(buf, 0, length) when creating the memory buffer for the whole message).